source/utilities.tex

```diff
@@ -2047,7 +2040,7 @@
 \indexlibrary{\idxcode{tuple_cat}}
 \begin{itemdecl}
 template <class... Tuples>
-  constexpr tuple<@\placeholder{CTypes}@...> tuple_cat(Tuples&&... tpls);
+  constexpr tuple<CTypes...> tuple_cat(Tuples&&... tpls);
 \end{itemdecl}
 
 \begin{itemdescr}
```

Comment:
The following description uses italic teletype *`T`_i* for elements of `Tuples`, which seems inconsistent with the changes made above.